AlfrescoApp/Images.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -87,6 +87,11 @@
       "idiom" : "ipad",
       "filename" : "app-icon-152.png",
       "scale" : "2x"
+    },
+    {
+      "idiom" : "ipad",
+      "size" : "83.5x83.5",
```

Comment:
This fixes the compiler warning, but we don't have the correctly-size icon file yet. I'll create a JIRA task to add one